tests/storage/export.go

```diff
@@ -155,7 +155,7 @@ var _ = SIGDescribe("Export", func() {
 	createDownloadPodForPvc := func(pvc *k8sv1.PersistentVolumeClaim, caConfigMap *k8sv1.ConfigMap) *k8sv1.Pod {
 		volumeName := pvc.GetName()
 		podName := "download-pod"
-		pod := tests.RenderPod(podName, []string{"/bin/sh", "-c", "sleep 360"}, []string{})
+		pod := tests.RenderPrivilegedPod(podName, []string{"/bin/sh", "-c", "sleep 360"}, []string{})
```

Comment:
Can manage with restricted here:
https://github.com/kubevirt/kubevirt/pull/8645/files#diff-4d9c88cecdb147d9936046660f590808a2eed4449c690dcc090b38b1b84e86f0R418-R420
(Just putting these here for future references instead of just private chat)